ynr/apps/uk_results/forms.py

```diff
@@ -107,7 +107,7 @@ def save(self, request):
             instance.save()
 
             winners = self.get_winners(num_winners=self.ballot.winner_count)
-            self.ballot.membership_set.update(elected=None)
+            self.ballot.membership_set.update(elected=False)
```

Comment:
As we are no longer allowing partial results, I feel we should explicitly set people as not elected, in the same way that previously we defaulted to `is_winner=False` on the CanidateResult. This stops lots of "Maybe" appearing the election result screen, where we have `None` values